src-plugins/medSegmentation/msegAlgorithmPaintToolbox.cpp

```diff
@@ -978,11 +981,7 @@ void AlgorithmPaintToolbox::updateWandRegion(medAbstractImageView * view, QVecto
     }
 
     if(!view->contains(m_maskAnnotationData))
-    {
         view->addLayer(m_maskAnnotationData);
-        setOutputMetadata(m_imageData, m_maskData);
-        medDataManager::instance()->importData(m_maskData);
-    }
```

Comment:
Removing this is not trivial and implies other changes.
Since it's not a big deal and I don't have much to think about it, I will leave this as is.